package.json

```
@@ -2,7 +2,7 @@
     "name": "nodecg-io",
     "private": true,
     "scripts": {
-        "postinstall": "cd nodecg-io-core && npm link && cd dashboard && npm link nodecg-io-core && npm i && cd ../../nodecg-io-twitch && npm link nodecg-io-core && npm link && cd ../sample && npm link nodecg-io-twitch nodecg-io-core && npm i",
+        "postinstall": "cd nodecg-io-core && npm link && cd dashboard && npm link nodecg-io-core && npm i && cd ../../nodecg-io-twitch && npm link nodecg-io-core && npm link && cd ../nodecg-io-rcon && npm link && cd ../sample && npm link nodecg-io-twitch nodecg-io-core nodecg-io-rcon && npm i",
```

Small error here. We will get rid of this mess with #7 soon anyway.
```suggestion
        "postinstall": "cd nodecg-io-core && npm link && cd dashboard && npm link nodecg-io-core && npm i && cd ../../nodecg-io-twitch && npm link nodecg-io-core && npm link && cd ../nodecg-io-rcon && npm link nodecg-io-core && npm link && cd ../sample && npm link nodecg-io-twitch nodecg-io-core nodecg-io-rcon && npm i",
```

---

Changed in https://github.com/codeoverflow-org/nodecg-io/pull/16/commits/71b44abec69963272d4dfb92558ca096d059dd9d